hbase-server/src/main/java/org/apache/hadoop/hbase/procedure2/store/region/RegionProcedureStore.java

```diff
@@ -552,11 +586,12 @@ public void insert(Procedure<?> proc, Procedure<?>[] subProcs) {
       for (Procedure<?> subProc : subProcs) {
         serializePut(subProc, mutations, rowsToLock);
       }
-      region.mutateRowsWithLocks(mutations, rowsToLock, NO_NONCE, NO_NONCE);
+      executeInsert(mutations, rowsToLock);
```

Comment:
Will this work?

We still might get another's rpc when the RpcServer is called because the row key clashes?

Reply:
Let me write a ut to verify this.